http/logical.go

```diff
@@ -39,12 +39,53 @@ func (b *bufferedReader) Close() error {
 	return b.rOrig.Close()
 }
 
-func buildLogicalRequestNoAuth(perfStandby bool, w http.ResponseWriter, r *http.Request) (*logical.Request, io.ReadCloser, int, error) {
+func buildLogicalPathNoAuth(r *http.Request) (string, int, error) {
```

Comment:
nit: This is purely fore building a logical path using the http request so rename it to be simply `buildLogicalPath`?